### PR TITLE
feat: v4.0.1

### LINKED ISSUE #98: "attribute-articles-singular" missing in lang/en.json
The error can be seen with the command `!monster -lang en`.

### LINKED ISSUE #100: locales-files contain wrong key for monster descriptions
The key "cmonster-more-descriptions" should be "cmonster-moredescriptions"

### PR DESCRIPTION
**Pull request by Stefouch**
### Added
- German translation for monsters and attacks for the Alien RPG.

### Changed
- (JavaScript) Drop of `utils/version.js` in favor of the version properties in `./package.json`.

### Fixed
- Resolves #100
- Resolves #98
- Arguments for `!attack` command if it's called from the `!monster` command.
- Incorrect key names in the language files.
- Missing keys in the german language file.

### PR Type

- [ ] This PR is a code change that implements a feature request.
- [x] This PR fixes an issue.
- [ ] This PR adds a new feature that is not an open feature request.
- [ ] This PR is not a code change (e.g. documentation, README, ...)
### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.

### CODE REVIEW

File: lang/de.json [hunk 1]
```diff
@@ -279,11 +279,15 @@
...
-            "• `dmg` | `damage` : Krtisiche Verletzungen durch Schaden.\n• `s`, `synth` | `synthetic` : Kritische Verletzungen von Androiden.\n• `x` | `xeno` : Kritische Verletzungen bei Xenomorphen.\n• `m` | `mental` : Dauerhafte mentale Traumata."
+            "• `dmg` | `damage` : Kritische Verletzungen durch Schaden.\n• `s`, `synth` | `synthetic` : Kritische Verletzungen von Androiden.\n• `x` | `xeno` : Kritische Verletzungen bei Xenomorphen.\n• `m` | `mental` : Dauerhafte mentale Traumata."
         ],
         [
             "🌟 Coriolis: Der Dritte Horizont",
             "• `dmg` | `damage` : Kritische Verletzungen durch Schaden.\n• `at` | `atypical` : Kritische Verletzungen durch atypischen Schaden."
+        ],
+        [
+            "🦋 Vaesen",
+            "• `dmg` | `damage` : Physical critical injuries.\n• `m` | `mental` : Mental critical injuries."
```
Comment (Stefouch) [inline lang/de.json:290]: @mprangenberg Could you add your translation to this line please ? :-)

Comment (mprangenberg) [reply lang/de.json:290]:
```suggestion
            "• `dmg` | `damage` : Kritische Verletzungen durch physischen Schaden.\n• `m` | `mental` : Kritische Verletzungen durch mentalen Schaden."
```

**Review by mprangenberg — DISMISSED** [review]
Looks good. I added a suggestion for the vaesen description for crit.

File: test/lang.test.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+			});
+
+			it(`Should have ${size} translation keys`, function() {
+				expect(Object.keys(locale)).to.have.lengthOf(size);
+			});
+
+			it('Should have all en.json keys', function() {
+				const out = [];
+				for (const key in en) if (!(key in locale)) out.push(key);
+				expect(out.join(), `Missing keys from ${lang}.json`).to.equal([].join());
```
Comment (mprangenberg) [inline test/lang.test.js:39]:
```suggestion
				expect([].join(), `Missing keys from ${lang}.json`).to.equal(out.join());
```

Comment (Stefouch) [reply test/lang.test.js:39]: Why this change ? I don't understand.

File: lang/de.json [hunk 3]
```diff
@@ -375,7 +375,7 @@
     "cmishap-invalid-reference": "Ungültige Magisches Mißgeschickreferenz",
     "cmodule-description": "Wirft die Würfel für ein Mechatron-Modul und prüft auf Überhitzung.",
     "cmonster-description": "Zeigt ein Monster aus den Katalogen an oder erzeugt ein zufälliges Monster anhand der Tabellen im *Zonenkompendium 1: Das Versteck der Reptiloiden* falls keine Argumente übergeben wurden.",
-    "cmonster-more-descriptions": [
```
Comment (Stefouch) [inline lang/de.json:372]: Found another one!

File: test/lang.test.js (hunk 2 above)
Comment (mprangenberg) [reply test/lang.test.js:39]: In my test the expected vs. actual output are in the wrong order. Expected would be empty and actual would be the expected key.

File: lang/de.json (hunk 3 above)
Comment (mprangenberg) [reply lang/de.json:372]: Yes, i also fixed that in my PR from earlier 😄

**Review by mprangenberg — DISMISSED** [review]
Looks good to me

**Pull request merged**